tripzone/src/containers/search/Search.js

```diff
@@ -0,0 +1,39 @@
+import React, { useContext, useEffect, useRef } from 'react'
+import styles from "./Search.module.scss"
+import useInput from '../../hooks/useInput';
+import AppContext from '../../context/appContext';
+import { getCityPromotion } from '../../utils/getCityPromotion';
+
+const Search = () => {
+
+    const [search, bindSearch] = useInput("");
+    const { setCityPromotion, setShowCityPromotion } = useContext(AppContext);
+
+    const inputReference = useRef(null);
+
+    useEffect(() => {
+        inputReference.current.focus();
+    }, [])
+
+    useEffect(() => {
+        async function fetchPromotion() {
+            if (search !== "") {
+                let promotionData = await getCityPromotion(search.toUpperCase())
+                setCityPromotion(promotionData);
+                setShowCityPromotion(true);
+            }
+        }
+        fetchPromotion();
+    }, [search]);
+
+    return (
+        <div className={styles.searchContainer}>
+            <h3>Travelling leaves you speechless, then turns you into a story teller</h3>
```

Comment:
could you let me know if tomorrow some change in heading is told, is it advisable to change in component level ?